frontend/src/containers/Homepage/Homepage.js

```diff
@@ -31,6 +33,12 @@ const Homepage = (props) => {
               <h5>Based on this week&apos;s popular ingredient donations</h5>
             </div>
           </div>
+          <div className="row">
+            <SearchResults searchTerm={searchTerm} />
+          </div>
+          <div className="row">
+            <SearchLozenge searchTerm={searchTerm} />
+          </div>
```

Comment:
Maybe we should consider embedding `SearchResults` and `SearchLozenge` into `RecipeSearchList`. That way we only have to fetch the search results once.

Reply:
Changes made!